### ISSUE TITLE
Unable to launch modpack

### ISSUE THREAD

**Issue by NRShao** [The-Nexus-Project/Limitless-3#203]
### Describe your issue, what happened?
When attempting to launch the pack from the GD and Curse launcher, the launch freezes on "constructing mods". It gets stuck on overwriting a file for some reason. I noticed that out of all the modpacks on my computer, the two files causing the freeze are .omniconfig files. (enigmaticlegacy-client/common.omniconfig) All other files are .toml. Maybe this is an issue?

### To Reproduce

Literally just downloaded the pack, and it will not launch on any launcher.

### Expected behavior
Modpack continues to load

### Screenshots
![Screen Shot 2021-06-17 at 10 00 14 PM](https://user-images.githubusercontent.com/14102516/122508786-67100500-cfb7-11eb-86ba-1c2461309fd7.png)

![Screen Shot 2021-06-17 at 9 55 00 PM](https://user-images.githubusercontent.com/14102516/122508330-ac800280-cfb6-11eb-9de1-00f81f418d6b.png)

### Crash Reports & Logs (Don't paste entire crash reports here.)
Log:
https://pastebin.com/kC0yP62v

### Affected Version
- Pack Version: 1.10.5c
- On a server?: No

**Comment by Gbergz** [comment]
I can see that you're on MACos, people have reported modpack not launching on it in the past and I've got no idea as to why unfortunately..

Edit: Try removing these configs `/enigmaticlegacy-client.omniconf` & `/config/enigmaticlegacy-client.omniconf`

**Comment by NRShao** [comment]
Yeah figured those were the problem. If I remove those, do I need to remove the modpack as well?

**Comment by Gbergz** [comment]
Maybe.. give it a try.

**Comment by NRShao** [comment]
So i gave it a shot and I deleted both the configs and the mod jar file. It just gets stuck on the previous step. Not sure what to look at to further diagnose :/

log:
https://pastebin.com/MuZRK2Tm

**Comment by Gbergz** [comment]
Yeah I'm at a loss when it comes to MAC issues, like I stated earlier, people on mac has had this issue in the past and I don't remember a solution. Very sorry.

**Comment by AdamBrianBright** [comment]
**TLDR;** Fixed by removing `Nicephore` mod (https://github.com/LotuxPunk/Nicephore)

Hi there! I was trying to launch this pack (version `1.10.6` on curseforge) via MultiMC on `MacOS BigSur 11.0.1` (intel chip) and faced the same problem.
I decided to compute the modlist difference against `ATM6 at the sky`, disabled all the different mods and enabled them in batch. It didn't take long, since without all of the different mods all worked fine and after re-enabling 10 of them it failed. So I disabled one by one until i found one mod that was causing this problem. After finding it, I re-installed pack from scratch and just disabled this particular mod... And it worked! 

I suggest to remove it from modlist, since it this mod is responsible purely for taking screenshots (it actually produces .exe file, which is not that good either). And if someone really need this mod, they can install it manually (like optifine lol)

**Comment by Gbergz** [comment]
@AdamBrianBright nice find!
I'll put a notice somewhere, or you can report it to LotuxPunk and try to get it fixed. I won't remove the mod.
People can just remove it manually.

**Comment by LotuxPunk** [comment]
Hi ! I was suggested it by Google, what a coincidence. I might take a look at it, wasn't aware it was an issue. (About infinite loading on MacOS)

@AdamBrianBright For the .exe producing, you can, always disable the optimization feature.
